src/theme/components/card.ts

```diff
@@ -119,8 +129,68 @@ const ReservoirCard = defineMultiStyleConfig({
           bodyMargin = "0";
         }
       }
+      // These sizes are only for the "row" layout.
+      // const imageSize = size ? imageSizes[size] : {};
       return {
         base: {
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
+            "[data-wrapper]": {
+              flexFlow: isRow ? "column nowrap" : null,
+            },
+            "[data-actions]": {
+              flexShrink: isAlignedRightActions ? "0" : null,
+              marginStart: "0",
+              marginTop: "xs",
+              maxWidth: "100%",
+            },
+            "[data-body]": {
+              width: "100%",
+            },
+            ...(isRow && {
+              "[data-image]": {
+                maxWidth: "100%",
+                margin: imageIsAtEnd ? "var(--nypl-space-m) 0 0" : null,
+                width: "100%",
+                ...imageSizesMinWidth0[imageSize],
+              },
+            }),
+          },
+          [`@container (min-width: ${breakpoints.sm})`]: {
+            "[data-body]": {
+              flexBasis: isRow ? "100%" : null,
+            },
+          },
+          [`@container (min-width: ${breakpoints.md})`]: {
```

Comment:
I wonder if this needs a +1 here. The following is the container query on the left and production on the right, setting the width to "md (600px)". I think we won't have parity but it's ok, I would rather use the breakpoint value than `{breakpoint} + 1`. Thoughts?

<img width="1299" alt="Screenshot 2025-01-29 at 9 27 40 PM" src="https://github.com/user-attachments/assets/e550529b-287a-4186-94cc-af616ab8abfb" />

Reply:
@EdwinGuzman I created a POC combining the container query and v4 feature branches. With the new breakpoint updates, I also reconfigured the values for the viewport sizing tool and using that tool make more sense now.

https://nypl-design-system-git-container-query-breakpoints-nypl.vercel.app/?path=/story/components-basic-elements-featuredcontent--with-controls

Reply:
Also, with the updated breakpoints, the breakpoints object includes semantic breakpoint names as well. `md` would be replaced with `mobile` or `largeMobile`.